tests/benchdnn/inputs/matmul/harness_matmul_smoke_ref

```diff
@@ -8,7 +8,7 @@
 --dtag=any
 
 --dt=f32,bf16,f16
---bia_dt=undef,f32
+--bia-dt=undef,f32
```

Comment:
(semi-random spot) it seems like there are a few more places in `tests/benchdnn/inputs/brgemm`, should we also update to not get a warning message?

Reply:
got it, thanks for clarification!